### DISCUSSION TITLE
scratch or temp disk on the planetary computer?

### DISCUSSION THREAD

**Discussion by aedessler** [microsoft/PlanetaryComputer#76] [General]
I'm guessing that the planetary computer hub compute servers we can spin up have a disk capacity of 15 GB.  Is there a temp or scratch area where we can write bigger files temporarily?  If not, that's something I would suggest be set up.  Lots of projects will generate intermediate work files that are much > 15 GB.

**Comment by TomAugspurger** [comment]
> Is there a temp or scratch area where we can write bigger files temporarily?

Yes, anything outside of `home/jovyan`. We recommend `/tmp/`. See https://planetarycomputer.microsoft.com/docs/overview/environment/#understanding-the-file-system for more.

For persisting large files beyond the lifetime of a single notebook server we recommend Azure Blob Storage (see https://planetarycomputer.microsoft.com/docs/quickstarts/storage/).
